precompiles/Playground.yul

```diff
@@ -925,6 +894,94 @@ object "Playground" {
                 c00, c01 := fp2Mul(a00, a01, g00, g01)
             }
 
+            // PAIRING FUNCTIONS
+            
+            function doubleStep(xq0, xq1, yq0, yq1, zq0, zq1) -> c00, c01, zero, zero, zero ,zero, c10, c11, c20, c21, zero, zero, c30, c31, c40, c41, c50, c51 {
+                let zero := ZERO()
+                let twoInv := montgomeryModularInverse(MONTGOMERY_TWO())
```

Comment:
Couldn't this value be precomputed?

Reply:
Indeed! Added here [6b2b1d4](https://github.com/lambdaclass/zksync_era_precompiles/pull/46/commits/6b2b1d473dbaac233b2899c193959b96e1007176)